Updated WP Plugin file and Project Root readme

ladland opened the pull request:
- Updated README.md to provide instructions to install deps.

- Updated nuxeo-plugin.php to fix 500 error due to unknown method on line 179

ladland committed:
Updated nuxeo-plugin.php

Fixed error on Line 179: removed unknown method getPrevious() as it is not a member of \Nuxeo\Automation\Client\Internals\NuxeoClientException or its base class

ladland committed:
README.md

Added Instruction to run composer update from root dir to pull required deps before upload

nuxeojenkins commented:
From the blame information on this pull request, potential reviewers: @pgmillon, @ldreuillat

Commit:
updated nx_object_list.twig to remove inline css and replaced it with css classes to allow for in template customisation

ladland committed:
updated README.md with CSS class names

pgmillon reviewed (changes requested):
Hi,
Thanks a lot for your interest in the Wordpress plugin !
The twig template improvement is great.
I'll fill the README in the next days to match with our template (http://doc.nuxeo.com/display/CORG/README+Template).
Can you remove + ignore the mac .DS_Store files ?

ladland committed:
Updated .gitignore to remove MacOS .DS_Store directories

Added nx-creator class to plugin-wp-nuxeo/resources/nx_object_list.twig to allow custom styles to be added to the Creator table entry

ladland commented:
The requested changes have been made

pgmillon commented:
I picked up your commits (you commited your wordpress folder) and updated the README.
